serverless.yml

```diff
@@ -0,0 +1,33 @@
+service: appsync-bootstrap-test
+provider:
+  name: aws
+custom:
+  accountId: 229055845648
+  appSync:
+    mappingTemplates:
+      - dataSource: Product
+        type: Mutation 
+        field: addProduct
+        request: put-request.txt
+        response: result-response.txt
+    serviceRole: "AppSyncServiceRole"
+    dataSources:
+      - type: AMAZON_DYNAMODB
+        name: Product
+        config:
+          tableName: Product-emulator
```

Comment:
This `.yml` file will actually be used to create tables in the amazon cloud, so its not necessary to add `-emulator` here.